cmd/fyne/internal/commands/mock_runner_test.go

```diff
@@ -76,26 +74,24 @@ func (t *testCommandCall) setEnv(env []string) {
 	sort.Strings(env)
 
 	// First check length of expected and passed environment are equal
-	assert.Equal(t.t, len(expectedEnv), len(env))
-	if len(expectedEnv) == len(env) {
-		// Check each independent environement variable match our expectation
-		for index, value := range env {
-			assert.Equal(t.t, expectedEnv[index], value)
-		}
+	require.Equal(t.t, len(expectedEnv), len(env))
+	// Check each independent environement variable match our expectation
+	for index, value := range env {
+		require.Equal(t.t, expectedEnv[index], value)
```

Comment:
Check all of env using `assert`

Reply:
If one is missing this is true - but what if one is incorrect... knowing if the others are also incorrect might be useful

Reply:
If one is incorrect the problem will happen very quickly as that will mess the sorting and the entire rest of the array might not match anymore (even if they do have the same number).